src/main/resources/balance.csv

```diff
@@ -0,0 +1,9 @@
+type,fruit,quantity
```

Comment:
add also file with report

Reply:
Actually, I added file with report in 'resources'.